src/pydantic_avro/base.py

```diff
@@ -40,6 +40,10 @@ def get_type(value: dict) -> dict:
             avro_type_dict: Dict[str, Any] = {}
             if "default" in value:
                 avro_type_dict["default"] = value.get("default")
+            if "description" in value:
+                avro_type_dict["doc"] = value.get("description")
+            if "allOf" in value and len(value["allOf"]) == 1:
```

Comment:
This was added for enums, frankly speaking, I do not know if cases with more allOf constraints arise, but this particular workaroud helps to handle the common case